searx/static/themes/simple/src/less/toolkit.less

```diff
@@ -339,6 +339,12 @@ select {
       border-bottom: 1px solid var(--color-search-border);
     }
   }
+
+  @media (prefers-color-scheme: dark) {
+    select {
+      background-image: data-uri('image/svg+xml;charset=UTF-8', '../svg/select-dark.svg');
```

Comment:
This is going to break, once the dark mode toggle gets implemented (or et least it would be a copy of the changes in definitions.less)... So I think it would be better to set the path as a var in `definitions.less`.. What do you think?

Reply:
`data-uri` is a less function that read and encode the file, so the CSS doesn't reference external files:
https://github.com/searxng/searxng/blob/4dd35f6d71d231334263365610775323166e9c88/searx/static/themes/simple/css/searxng.css#L1705-L1709

So it is not possible to use a CSS variable, but a LESS variable is possible.

Reply:
Yeah that is a great idea, thank you! I just think its better to keep the dark/light and color specific stuff in `definitions.less`. So adding a less var with the path to the select depending on dark/light would be the better option I believe... (If that works tho; cannot test it right now sry)

Reply:
> ] so the idea is less generates the prefer dark theme media query automatically, since the are two different values in the svg path less variable, but idk if that actually works tho

I'm not sure how to do that.

Looking at https://lesscss.org/features/#variables-feature-variable-interpolation it is possible to have

```less
@{dark-selector} {
   ....
}
```

but `dark-selector: @media (prefers-color-scheme: dark), html.dark-theme` doesn't work.

The only way I see is two less variables and copy / paste the definitions.
This is what postcss does: https://equinusocio.dev/blog/making-dark-theme-switcher-with-postcss/ (see the "Where is the magic?" section).

A workaround to avoid to rewrite everything ( = to keep less), we can use either a mixin or even a dedicated file (`dark.less`).

Reply:
I'm going to merge this PR, we can adjust the dark mode definition later.